src/components/Local.vue

```diff
@@ -98,12 +99,19 @@ export default {
   },
 
   methods: {
+    getNow() {
+      const today = new Date();
+      const month = today.toLocaleString("default", { month: "long" });
+      const date = today.getDate() + " " + month + ", " + today.getFullYear();
+      return date + " " + today.getHours() + ":" + today.getMinutes();
+    },
```

Comment:
thank you for showing me how to use the date in JS  :-) , I always found it intemedating !!

Reply:
Actually I also googled for this and I mixed two answers from the Stackoverflow to achieve my desired date format. :smile:

Reply:
I know those stackoverflow searches right !!